app/views/admin/infobase/_registered_users.html.haml

```diff
@@ -0,0 +1,14 @@
+%table.ui.very.basic.table{style: 'border: none'}
+  %thead
+    %tr
+      %th Name
+      %th E-mail
+      %th Date of Registration
+      %th.right.aligned Status
+  %tbody
+    - @users.each do |user|
+      %tr
+        %td{"data-label" => "Name"}= user.name
+        %td{"data-label" => "E-mail"}= user.email
+        %td{"data-label" => "Date of Registration"}= user.created_at.to_date
```

Comment:
until `.to_date` it was fine, as soon as `.to_date` appears - straight to the decorator
also, I forgot to mention, we used to have decorators for almost every model in the app, so don't be scared if you will end up like that